op-program/client/interop/consolidate.go

```diff
@@ -157,13 +158,19 @@ type ConsolidateCheckDeps interface {
 	cross.UnsafeStartDeps
 }
 
-func checkHazards(
-	deps ConsolidateCheckDeps,
-	candidate supervisortypes.BlockSeal,
-	chainID eth.ChainID,
-	execMsgs []*supervisortypes.ExecutingMessage,
-) error {
-	hazards, err := cross.CrossUnsafeHazards(deps, chainID, candidate, execMsgs)
+func checkHazards(logger log.Logger, deps ConsolidateCheckDeps, candidate supervisortypes.BlockSeal, chainID eth.ChainID, execMsgs map[uint32]*supervisortypes.ExecutingMessage) error {
+	// TODO(#14234): remove this check once the supervisor is updated handle msg expiry
+	messageExpiryTimeSeconds := deps.DependencySet().MessageExpiryWindow()
+	for _, msg := range execMsgs {
+		if msg.Timestamp+messageExpiryTimeSeconds < candidate.Timestamp {
+			return fmt.Errorf(
+				"message timestamp is too old: %d < %d: %w",
+				msg.Timestamp+messageExpiryTimeSeconds, candidate.Timestamp, supervisortypes.ErrConflict,
+			)
+		}
+	}
```

Comment:
conflict with develop. That TODO has been fixed as message checks are now handled by the supervisor logic. So we don't need to do this here.